tests/unit/test_dbapi__helpers.py

```diff
@@ -71,6 +72,33 @@ def test_scalar_to_query_parameter(self):
             self.assertEqual(named_parameter.type_, expected_type, msg=msg)
             self.assertEqual(named_parameter.value, value, msg=msg)
 
+    def test_decimal_to_query_parameter(self):  # TODO: merge with previous test
```

Comment:
I should have removed that, it's just a subset of the `test_scalar_to_query_parameter` test above it.